setup.py

```diff
@@ -0,0 +1,63 @@
+"""
+Setup file
+"""
+
+# To use a consistent encoding
+import codecs
+import os
+
+# Always prefer setuptools over distutils
+from setuptools import setup, find_namespace_packages
+
+
+def get_long_description():
+    """
+    Get the long description from the README file
+    """
+    here = os.path.abspath(os.path.dirname(__file__))
+
+    with codecs.open(os.path.join(here, 'README.md'), encoding='utf-8') as readme:
+        return readme.read()
+
+
+setup(
+    name='cli',
+    version='0.0.1',
+    description='packaging wrapper using ansible',
+    long_description=get_long_description(),
+    long_description_content_type='text/markdown',
+    url='https://github.com/theforeman/foreman-quadlet',
+    author='The Foreman Project',
+    author_email='foreman-dev@googlegroups.com',
+    zip_safe=False,
+
+    classifiers=[
+        'Development Status :: 3 - Alpha',
+        'Intended Audience :: Developers',
+        'Topic :: Software Development :: Build Tools',
+        'License :: OSI Approved :: GNU General Public License v2 (GPLv2)',
```

Comment:
Can you add `license='GPL-2.0-only'` to `setup()` and drop this line to avoid the deprecation warning?
```suggestion
```

I did the same in obsah: https://github.com/theforeman/obsah/commit/3c65e0ac0f8f75c31cb1159fcc13f7f54b538d12 added the `license` and https://github.com/theforeman/obsah/commit/52808215031233b63c081e953582cd7ac6661d30 dropped the classifier